src/components/BookCard.jsx

```diff
@@ -135,6 +199,16 @@ function BookCard({ user_uid, image, genres, title, price, rating, id }) {
             <></>
           )}
         </div>
+        <div className="flex justify-start items-center">
+          <ReactStars
+            className=""
+            size={20}
+            isHalf={true}
+            value={currentRating}
+            onChange={(newR) => setRating(newR)}
+          />
+          <span className="pl-1 mt-1">{currentRating}(5)</span>
```

Comment:
No need to add this 👉🏼 "(5)", we already have just 5 stars so the user knows it's whatever/5

Reply:
Ok